Userland/Utilities/sed.cpp

```diff
@@ -87,8 +87,12 @@ static ErrorOr<SubstitutionCommand> parse_command(StringView command)
             if (flag_filepath.is_empty())
                 return Error::from_string_literal("No filepath was provided for the 'w' flag.");
             output_filepath = flag_filepath;
-        } else
+        } else if (flag == "g"sv) {
+            // Allow multiple matches per line
+            options &= ~PosixFlags::SingleMatch;
```

Comment:
This bit magic is hard to read, can you rewrite this another way?

Reply:
I rewrote the comment to hopefully make it clearer, but I honestly don't know of a cleaner way to un-set one bit of a bitfield. Happy to revise this if you or anyone else has a suggestion!